src/orca/services/nextflowtower/client.py

```diff
@@ -113,7 +113,9 @@ def request_paged(self, method: str, path: str, **kwargs) -> dict[str, Any]:
         while num_items < total_size:
             kwargs["params"]["offset"] = num_items
             json = self.request_json(method, path, **kwargs)
-            total_size = json.pop("totalSize")
+            total_size = (
```

Comment:
This change and the one just below were made because the `tasks` API endpoint response JSON uses `total` while the paginating endpoints we were already using have `totalSize`.

Reply:
if `json.get("totalSize")` is 0 it'll do a `json.pop("total")`. I'm not sure if that is intended in this case.

Reply:
The response `json` will have either `"totalSize"` or `"total"`, not both. Those are the cases I was trying to allow for.

Reply:
If they have either or - this would throw a KeyError if "totalSize" = 0 since "total" is not present

Reply:
addressed in latest commit

Reply:
I think chaining .get would work:
```
json = {
    "totalSize": 1,
    "total": 2
}

# Print 1
print(json.get("totalSize", json.get("total", 0)))

json = {
    "total": 2
}

# Print 2
print(json.get("totalSize", json.get("total", 0)))

json = {}

# Print 0
print(json.get("totalSize", json.get("total", 0)))
```

Reply:
Chaining .get actually breaks a whole bunch of tests, I guess either it doesn't like the default 0 or the `pop` removing the dictionary key is needed

Reply:
What about 
`json.pop("totalSize", None) or json.pop("total", 0)`